dspace-api/src/test/java/org/dspace/content/TiltedRelationshipMetadataServiceIT.java

```diff
@@ -0,0 +1,97 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.content;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import java.util.List;
+
+import org.dspace.builder.EntityTypeBuilder;
+import org.dspace.builder.ItemBuilder;
+import org.dspace.builder.RelationshipBuilder;
+import org.dspace.builder.RelationshipTypeBuilder;
+import org.junit.Test;
+
+/**
+ * This class carries out the same test cases as {@link RelationshipMetadataServiceIT} with a few modifications.
+ */
+public class TiltedRelationshipMetadataServiceIT extends RelationshipMetadataServiceIT {
+
+    /**
+     * Call parent implementation and set the tilted property of {@link #isAuthorOfPublicationRelationshipType}.
+     */
+    @Override
+    protected void initPublicationAuthor() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        EntityType publicationEntityType = EntityTypeBuilder.createEntityTypeBuilder(context, "Publication").build();
+        EntityType authorEntityType = EntityTypeBuilder.createEntityTypeBuilder(context, "Author").build();
+        leftItem = ItemBuilder.createItem(context, col).withRelationshipType("Publication").build();
+        rightItem = ItemBuilder.createItem(context, col).withRelationshipType("Author")
+                .withPersonIdentifierLastName("familyName")
+                .withPersonIdentifierFirstName("firstName").build();
+        isAuthorOfPublicationRelationshipType =
+                RelationshipTypeBuilder.createRelationshipTypeBuilder(context, publicationEntityType, authorEntityType,
+                        "isAuthorOfPublication", "isPublicationOfAuthor",
+                        null, null, null, null).build();
+
+        isAuthorOfPublicationRelationshipType.setTilted(RelationshipType.Tilted.LEFT);
```

Comment:
I'm wondering we don't have a similar test for a `Tilted.RIGHT` relationship?  It seems like we should be testing both values of `Tilted` (though we don't need any for `Tilted.NONE` since that's the default and is tested by all other tests).